doc/whats_new/v1.6.rst

```diff
@@ -182,6 +182,10 @@ Changelog
   estimator without re-fitting it.
   :pr:`29067` by :user:`Guillaume Lemaitre <glemaitre>`.
 
+- |Fix| :class:`model_selection.RepeatedStratifiedKFold` has the function split,
+  where 'y' parameter is made as mandatory input field.
```

Comment:
Maybe something like this? Not 100% sure about the renderin please check the rendered doc in the "check the rendered doc" CI status
```suggestion
- |Fix| Improve error message when :func:`model_selection.RepeatedStratifiedKFold.split` is called without a `y` argument
```